### DISCUSSION TITLE
Turn on vase mode after certain layer height

### DISCUSSION THREAD

**Discussion by GlauTechCo** [OrcaSlicer/OrcaSlicer#4951] [Ideas]
Hello, when we think of a smart vase that will hide the electronic circuits at the bottom, can a feature be added to print the bottom layers normally and the vase part with vase mode?

<img width="338" alt="Ekran Resmi 2024-04-10 11 33 31" src="https://github.com/SoftFever/OrcaSlicer/assets/33813227/74157542-859b-4a87-bdac-0ec510526d06">
<img width="527" alt="Ekran Resmi 2024-04-10 11 34 33" src="https://github.com/SoftFever/OrcaSlicer/assets/33813227/262aac0c-d51e-46e2-a967-1807728a5ef5">

**Comment by PeterArbeitsfrei** [comment]
Yea I also would like this feature, so far I was only able to do it by creating two seperate gcode files and combining them, obviously that's a pretty complex way of doing it but it works really good and shows that a mode where the slicer simply change to vase mode also wouldn't need to be too complicated.

**Reply by bogdanvarlamov** [reply]
@Bobylein you slice up the bottom first and then save the gcode, and then slice up the "vase" part, save that gcode, and then paste the vase to the end of the bottom plate gcode?

**Comment by KKomarov** [comment]
Common case for me when you need some complex bottom and seamless shell on top of it. For silicone casting or joining two vases by bottoms into one two sided vase mold.
